controllers/users.js

```diff
@@ -315,6 +315,35 @@ const verifyUserImage = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const updateDiscordUserNickname = async (req, res) => {
+  try {
+    const userToBeUpdated = await dataAccess.retrieveUsers({ id: req.params.userId });
```

Comment:
We can save this as a variable and use it 
`const {userId}=req.params`

`

     return res.json({
       userAffected: {
         userId
         username: userName,
         discordId,
       },
       message: response,
     });
`